hard/forth.go

```diff
@@ -0,0 +1,54 @@
+package hard
+
+import (
+	"strconv"
+	"strings"
+)
+
+func Forth(input string) int {
+	stack := []int{}
+	words := strings.Fields(input)
+	definedWords := make(map[string]string)
+
+	for i := 0; i < len(words); i++ {
+		word := words[i]
+		switch strings.ToUpper(word) {
+		case "+":
```

Comment:
Could you please format the 'rusky code' and add appropriate spacing in the case statement? This will enhance its readability and understanding.

Reply:
Yeah was weird at first but when i manually do that and do save eslint auto place that in that position so i guess this is the correct way.